packages/tools/src/printCLIPath.ts

```diff
@@ -0,0 +1,13 @@
+import path from 'path';
+import pkg from '@react-native-community/cli/package.json';
```

Comment:
This assumes `cli-tools` has a dependency on `cli`, which is not true. Moving `printCLIPath` to `cli` would solve this problem

Reply:
I forgot to mention this creates a circular dep, because cli relies on cli-tools

Reply:
We can move it to the CLI, no problem. It will be actually even a bit more simple.

Reply:
Implemented alternative solution that doesn't rely on package.json.